contract-builder/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM rust:1.69.0
+FROM rust:1.72.1
```

Comment:
nearcore recently bumped their MSRV to 1.73, which will hit us with the next near-primitives release, so I guess we can already bump it to 1.73.